src/binary/ibc-relayer/commands/start.ts

```diff
@@ -24,15 +26,18 @@ type Flags = {
   mnemonic?: string;
   srcConnection?: string;
   destConnection?: string;
+  poll?: number;
+  maxAgeSrc?: number;
+  maxAgeDest?: number;
 } & LoggerFlags;
```

Comment:
`Partial<LoopOptions>` could be used here instead.

I'm going to clean this up anyway so we can merge.